addons/stock/models/res_company.py

```diff
@@ -44,6 +44,10 @@ def _default_confirmation_mail_template(self):
         help="""Day of the month when the annual inventory should occur. If zero or negative, then the first day of the month will be selected instead.
         If greater than the last day of a month, then the last day of the month will be selected instead.""")
 
+    # used for sending stock text confirmation
+    stock_text_confirmation = fields.Boolean("Stock Text Confirmation", default=True)
+    stock_confirmation_type = fields.Selection([('sms', 'SMS')], default='sms')
```

Comment:
Would say that field should exist with 'none' only in base module, then sub-modules (sms, whatsapp) add their new key. It is effectively a good idea to have a modular approach (base field with additional options in sub modules), but it is weird to talk about SMS in base stock. It should be some "phantom" field in stock, then sub modules make it appear and usable.